docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/arithmetic/segment-max-16.rst

```diff
@@ -0,0 +1,108 @@
+SegmentMax
+==========
+
+
+.. meta::
+  :description: Learn about SegmentMax-16 - an arithmetic operation which computes the maximum values along segments of a tensor.
+
+**Versioned name**: *SegmentMax-16*
+
+**Category**: *Arithmetic*
+
+**Short description**: *SegmentMax-16* operation finds the maximum value in each specified segment of the ``input`` tensor.
+
+**Detailed description**
+
+For each index in ``segment_ids`` the operator gets values from ``data`` input tensor and calculates the maximum value for each segment.
+
+For example ``segments_ids`` with value ``[0,0,0,1,1,3,5,5]`` defines 4 non-empty segments. When coupled with a 1D data tensor ``data``, the segments are as follows:
+
+* Segment_0: ``[data[0], data[1], data[2]]``
+* Segment_1: ``[data[3], data[4]]``
+* Segment_2: ``[]``
+* Segment_3: ``[data[5]]``
+* Segment_4: ``[]``
+* Segment_5: ``[data[6], data[7]]``
+
+When there are no values in a segment, ``output[segment]`` is defined by ``fill_mode`` attribute.
+
+For ``fill_mode`` equal to ``ZERO`` , the operation output would be ``[max(Segment_0), max(Segment_1), 0, max(Segment_3), 0, max(Segment_5)]``.
+
+**Attributes**:
+
+* **1**: *empty_segment_value*
+
+  * **Description**: The value assigned to segments which are empty. **Required.**
+  * **Range of values**: Name of the mode in string format:
+
+    * ``ZERO`` - the empty segments are filled with zeros.
+    * ``LOWEST`` - the empty segments are filled with the lowest value of the data type *T*.
+  * **Type**: ``string``
+
+**Inputs**
+
+* **1**: ``data`` - ND tensor of type *T*, the numerical data on which SegmentMax operation will be performed. **Required.**
+
+* **2**: ``segment_ids`` - 1D Tensor of sorted non-negative numbers of type *T_IDX1*. Its size is equal to the size of the first dimension of the ``data`` input tensor. The values must be smaller than ``num_segments``. **Required.**
+
+* **3**: ``num_segments`` - A scalar value of type *T_IDX2* representing the segments count, used for shape inference. **Optional.**
+
+**Outputs**
+
+* **1**: The output tensor has same rank and dimensions as the ``data`` input tensor except for the first dimension which is calculated as ``max(segment_ids) + 1`` 
```

Comment:
``max(segment_ids) + 1`` is not true always. If `num_segments` is specified it will be equal to `num_segments`

Reply:
Fixed, waiting with a push for the first comment to be resolved.